test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/MessagePackHubProtocolTests.cs

```diff
@@ -43,6 +43,8 @@ public class MessagePackHubProtocolTests
             new object[] { new[] { new StreamItemMessage("xyz", new CustomObject()) } },
             new object[] { new[] { new StreamItemMessage("xyz", new[] { new CustomObject(), new CustomObject() }) } },
 
+            new object[] { new[] { new CancelInvocationMessage("xyz ") } },
```

Comment:
space in id needed?

Reply:
this is just another character in the invocation id... 

![image](https://user-images.githubusercontent.com/1438884/31730907-36be2588-b3e8-11e7-92f2-6305180952c2.png)